doc/ledger3.texi

```diff
@@ -512,7 +512,7 @@ Ledger will generate:
 
 @smallexample @c output:66E3A2C
 10-Dec-01 Checking balance      Assets:Checking          $ 1,000.00   $ 1,000.00
-                                Equit:Opening Balances  $ -1,000.00            0
+                                Equity:Opening Balances  $ -1,000.00            0
```

Comment:
Hi @jostber,
as @thdox already stated the proposed changes break the ledger validation.

Please take a moment to read the rather lengthy (but very worthwhile ;) comment at the beginning of [`doc/ledger3.texi`](https://github.com/ledger/ledger/blob/master/doc/ledger3.texi#L23-L93) it sheds some light on how examples in the documentation are validated
to ensure that the given examples do actually work with ledger.

There are different ways of running the DocTests:
* `python test/DocTests.py -vv --ledger ${PATH_TO}/ledger --file ${PATH_TO}/doc/ledger3.texi`
* `ctest --output-on-failure -R DocTests` (which just runs the above command for you)